src/main/java/com/conveyal/datatools/manager/models/transform/NormalizeFieldTransformation.java

```diff
@@ -198,10 +198,18 @@ public void transform(FeedTransformZipTarget zipTarget, MonitorableJob.Status st
             Field[] fieldsFoundInZip = gtfsTable.getFieldsFromFieldHeaders(headers, null);
             int transformFieldIndex = getFieldIndex(fieldsFoundInZip, fieldName);
 
+            // If the index is -1, this is a new column, and we need to add it accordingly
+            if (transformFieldIndex == -1) {
```

Comment:
These two new sections were `if (transformFieldIndex == -1)` are very similar. Could they be moved into a separate method?

Reply:
It might be cleaner, but it'll be more lines and the shared method potentially quite confusing with the types. What do you think?

Reply:
The number of lines will be the same. If the method is generic I don't think it would be confusing:

    private String[] expandArray(String[] array, String value) {
        String[] expanded = new String[array.length + 1];
        System.arraycopy(array, 0, expanded, 0, array.length);
        expanded[array.length] = value;
        return expanded;
    }

and then use this, like:

    writer.write(expandArray(headers, fieldName));
    writer.write(expandArray(csvValues, transformedValue));